webpack/prod.js

```diff
@@ -23,4 +26,9 @@ module.exports = merge(base, {
       }),
     ],
   },
+  plugins: [
+    new CopyPlugin({
+      patterns: [{ from: "./src/assets", to: "src/assets" }],
```

Comment:
This should be from `./src/assets` to `./dist/src/assets`, since we are only keeping the `dist` folder after build.